includes/class-amp-story-post-type.php

```diff
@@ -323,6 +329,20 @@ function( $sanitizers ) {
 			}
 		);
 
+		add_filter(
+			'amp_content_sanitizers',
+			function( $sanitizers ) {
+				if ( is_singular( self::POST_TYPE_SLUG ) && isset( $_GET['story_export'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Should this include a cap check as well? Otherwise, anyone would be able to make this request. 

Actually, since the request may not be authenticated, perhaps doing the nonce check here is what is more appropriate.

Reply:
In other words, add this as a condition:

```php
check_ajax_referer( 'amp-story-export', 'nonce', false )
```

So:

```diff
- if ( is_singular( self::POST_TYPE_SLUG ) && isset( $_GET['story_export'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+ if ( is_singular( self::POST_TYPE_SLUG ) && isset( $_GET['story_export'] ) && check_ajax_referer( 'amp-story-export', 'nonce', false ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```